README.md

```diff
@@ -17,11 +17,16 @@ To run a single benchmark, run: `./run_single_bench.sh <BENCHMARK_NAME>` .
 To clean-up the test images, run: `./clean.sh` .
 
 ## Configuring the benchmark
-Test duration is governed by environmental variable `GRPC_BENCHMARK_DURATION`. Defaults to `30s`.
-
-The maximum number of cpus used by server is governed by the environmental variable `GRPC_SERVER_CPUS`. Defaults to `1`.
-
-Which tests to run is governed by a regex in `GRPC_BENCHMARK_SELECT`, for example `rust` or `(kotlin|java)`. Defaults to all tests.
+The benchmark can be configure through the following environment variables:
+
+|**Name**|**Description**|**Default value**|
+|--------|---------------|:---------------:|
+|GRPC_BENCHMARK_DURATION|Duration of the benchmark.|30s|
+|GRPC_SERVER_CPUS|Maximum number of cpus used by the server.|1|
+|GRPC_BENCHMARK_SELECT|Which tests to run. For example `rust` or `(kotlin|java)`.|. (*all tests*)|
```

Comment:
There is no more GRPC_BENCHMARK_SELECT on master after my change. Now we can either specify the list of benchmarks to run: `./bench.sh bench1 bench2...` or use shell patterns, i.e. `./bench.sh java* rust* cpp*`

Reply:
Yeah I'll need to do some good old `rebasing`